osu.Game.Rulesets.Osu.Tests/SpinFramesGenerator.cs

```diff
@@ -14,7 +14,7 @@ public class SpinFramesGenerator
         /// <summary>
         /// A small amount to spin beyond a given angle to mitigate floating-point precision errors.
         /// </summary>
-        public const float SPIN_ERROR = MathF.PI / 8;
+        public const float SPIN_ERROR = 1;
```

Comment:
I'm confused. $\frac{\pi}{8}$ radians is about 0.4 in decimal, or 45 degrees. This change is changing the value but seemingly not the unit? And one radian is about 60 degrees? As such, while the PR description talks about "reducing the error margin", this seems to... increase it?

Reply:
Okay I have no idea about this test then. Probably just `[Ignore]` as "no idea why it's off". It's off even in forwards playback, and even when the visual cursor position looks correct.